internal/job/shell/logger.go

```diff
@@ -91,6 +96,15 @@ func (wl *WriterLogger) Warningf(format string, v ...any) {
 	wl.Printf("^^^ +++")
 }
 
+func (wl *WriterLogger) OptionalWarningf(id, format string, v ...any) {
+	if slices.Contains(wl.DisabledWarningIDs, id) {
+		return
```

Comment:
This is what I came to check - whether or not you downgrade to `info` level log line, or drop it entirely. Perhaps a `debug` line is a good middle ground? discoverable if debug is enabled, otherwise not?

Reply:
ah, logs don't quite work that way (with levels etc) in the shell, they're either there or they're not - i think we should probably downgrade it to just a call to `wl.Printf` though.

Reply:
Ah, okay. Yeah, I think printing something is better than nothing, and if that itself proves to be a pain point for people, we can evaluate dropping them entirely then